src/app/utils/objectToQueryString.ts

```diff
@@ -0,0 +1,8 @@
+const objectToQueryString = (obj: { [key: string]: unknown }) => {
```

Comment:
Why don't you use `Selects` type here?

Reply:
Because I want this function be more flexible, not only for objects like Selects.

Reply:
Then use `Record<>` for an object type